src/pages/resource/ResourcePage.tsx

```diff
@@ -56,11 +71,23 @@ const ResourcePage: FC<RouteProps> = (props) => {
   }, [identifier]);
 
   return (
-    <StyledPageContent>
-      {isLoadingResource && <CircularProgress />}
-      {resourceLoadingError && <ErrorBanner />}
-      {resource && <ResourcePresentation resource={resource} />}
-    </StyledPageContent>
+    <>
+      <StyledResourceActionBar>
+        <Button
+          size="large"
+          variant="outlined"
+          color="primary"
+          data-testid={`edit-resource-button`}
+          onClick={handleClickEditButton}>
+          {t('common.edit').toUpperCase()}
+        </Button>
```

Comment:
Skal denne komponenten være gjenbrukbar for all ressursvisning, ikke bare for eieren av ressursen? I så fall bør man kanskje ha noen sjekker på om brukeren får redigere ressursen før de får se knappen om å redigere.

Reply:
du har rett. også glemt å legge inn sjekk på om published